client/src/modules/templates/bhInventorySelect.tmpl.html

```diff
@@ -0,0 +1,24 @@
+<div ng-form="{{ $ctrl.name }}" bh-inventory-select ng-model-options="{ updateOn: 'default' }">
+  <div
+    class="form-group"
+    ng-class="{ 'has-error' : InventoryForm.$$parent.$submitted && InventoryForm.inventory_id.$invalid }">
```

Comment:
Where is `InventoryForm` defined?

I think you can probably remove the `ng-form = "{{ $ctrl.name }}"` call and just use `ng-form="InventoryForm"`.  That should work well and eliminate need for the `name` parameter in the `bhInventorySelect` component :+1: